influxdb/src/error.rs

```diff
@@ -1,8 +1,9 @@
 //! Errors that might happen in the crate
 
+use reqwest::StatusCode;
```

Comment:
This use needs to be feature-gated, and you need to add a separate import when `surf` is used instead of `reqwest`

Reply:
Yes, my bad, will look into it! Thanks for the remarks

Reply:
@0xDmtri please note #147, but @msrd0's comment stands

Reply:
Current plan was to support `surf` for one more release to give people time to respond to #147

Reply:
I have changed reqwest::StatusCode to http::StatusCode